interlay/interbtc-ui
# fix(Loans): withdraw action

# Interbtc UI Pull Request Template

## Description

Close https://github.com/interlay/interbtc-ui/issues/935

## Reproducible testing steps:

- Open Lend position and turn it into collateral
- Open Borrow position
- Try withdraw max limit
- Should be able to withdraw that amount
